### PR TITLE
Initial draft of MAINTAINERS.md

### PR DESCRIPTION
**Pull request by jmertic**
Needs TSC approval before merging.

### CODE REVIEW

**Comment by rajddesai** [comment]
Approve

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# ODPi Maintainers
+## What is an ODPi Maintainer?
+
+ODPi Maintainers are the key leaders in the development of the ODPi specifications. They are key members of the ODPi community who have shown both leadership and collaboration skills in thier work within the community.
```
Comment (alanfgates) [inline MAINTAINERS.md]: typo, thier should be their

File: MAINTAINERS.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+### Roles and Responisibilities
+
+* Review spec and test harness contributions that for the spec(s) you are a maintainer for.
```
Comment (alanfgates) [inline MAINTAINERS.md]: This sentence is incoherent.  I think it means, "Review spec and test harness contributions for the spec(s) you are a maintainer for.", but I'm not 100% certain that's the intent.

File: MAINTAINERS.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+* Control the direction of the spec(s) you are a maintainer for.
+* Act as the point-of-contact for questions from users and other committers for the spec(s) you are a maintainer for.
+* Adhere to and be an example of the [ODPi Code of Conduct](https://github.com/odpi/specs/wiki/ODPi-Code-of-Conduct)
+* Elect ODPi development community members to the TSC and RT
+
+## How to become an ODPi Maintainer?
+
+ODPi Maintainers are elected by the other ODPi Maintainers. There is no membership requirement with ODPi to become an ODPi Maintainer.
```
Comment (alanfgates) [inline MAINTAINERS.md]: Are maintainers elected at a set time (like TSC and RT) or is it done anytime an existing maintainer proposes electing a new maintainer?  I would assume the latter.

**Comment by rvs** [comment]
It looks like @rajddesai approved it. Is there anything else holding up the merge?

**Comment by jmertic** [comment]
Waiting to get the final list of maintainers cleaned up. Should be ready to post 9/30

**Comment by jmertic** [comment]
Going to pull in for now and add in the ODPi Maintainers later.

**Pull request merged**